src/components/note.js

```diff
@@ -1,12 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Note = ({ note }) => (
-  <div>
-    <h2><small>{note.title}</small></h2>
-    <div>{note.content}</div>
-  </div>
-);
+class Note extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.handleOnchangeNote=this.handleOnchangeNote.bind(this);
```

Comment:
Using arrow function auto binding instead of bind context. More detail at [here](https://reactjs.org/docs/react-without-es6.html#autobinding)